model/registry/mobile-events.yaml

```diff
@@ -0,0 +1,78 @@
+groups:
+  - id: registry.ios.lifecycle.events
+    prefix: ios
```

Comment:
I think we should keep the structure in the registry consistent, with the "rule" that the `yaml` file is named by the top-level namespace.

If we start mixing namespace based naming with semantic naming / grouping (here: `mobile-events`) it will impact discoverability. Same is true (maybe even more important) for the corresponding markdown files and structure.

@joaopgrassi does it make sense?

Reply:
Yes!

Reply:
@AlexanderWert fixed! `android` already existed, so I just renamed what I had to `ios` and moved the android ones to the exist. PTAL, thanks!